### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by acka47**
Enjoy!

### CODE REVIEW

**Comment by literarymachine** [comment]
@acka47 it looks to me like this needs some work in Crowdin, especially the translation of `about.md` after its recent restructuring.

**Comment by gandalfar** [comment]
It looks to me that this PR needs to be rebased, before it can get merged.

I'm also worried that it seems to replace a lot of German translations with English texts. Maybe it would be good if @trugwaldsaenger or someone else could review German translation in Crowdin.

I also wonder who is owner of PT language translation and if they could also review translations in Crowdin.

**Comment by literarymachine** [comment]
> It looks to me that this PR needs to be rebased, before it can get merged.

Yes, I can take care of merging the conflict in package-lock.json

> I'm also worried that it seems to replace a lot of German translations with English texts. Maybe it would be good if @trugwaldsaenger or someone else could review German translation in Crowdin.

This is the much bigger issue here. I am not sure what has gone wrong, some documents might need a complete re-translation.

> I also wonder who is owner of PT language translation and if they could also review translations in Crowdin.

That would be @tamielbr!

**Comment by tamielbr** [comment]
@gandalfar I can review. I hope it's not the whole thing ;) If you can point me to what needs review (or if its already flagged) I can do it.

**Comment by acka47** [comment]
>>   I'm also worried that it seems to replace a lot of German translations with English texts. Maybe it would be good if @trugwaldsaenger or someone else could review German translation in Crowdin.
>
> This is the much bigger issue here. I am not sure what has gone wrong, some documents might need a complete re-translation. @acka47 used to be in charge of the Crowdin integration, not sure if he still is.

This is the old annoying CrowdIn problem.

1.) CrowdIn behaves as the primary language data source: You can not change language files in GitHub so that CrowdIn takes over the changes but it will always overwrite changes with the previous strings.
2.) Crowdin does not cope well with additional lines in the source language (here English): If you add a paragraph in between, it just sets all the other languages back to the source language. Then you have to go into CrowdIn and translate those strings once again (at least it already suggest the translations and you just have to do a lot of click work).

To sum it up, somebody has to do the click work in CrowdIn. This time, I will do it once more.

**Comment by gandalfar** [comment]
@acka47 thanks for handling that. Could you please also document this process in a non-technical way (with clear steps / screenshots to take). This way it's going to be easier for us to find non-technical person to do it next.

Maybe it's already documented somewhere, but I don't see it linked from README (in main or -ui repo).

**Comment by acka47** [comment]
I updated the German strings that already were translated in the past in CrowdIn but there are still lots that haven't been translated to German yet, especially those relatively new wrt policies.

I didn't touch Portuguese, though, but after a short look it seems ok.

> Could you please also document this process in a non-technical way (with clear steps / screenshots to take). This way it's going to be easier for us to find non-technical person to do it next.
>
> Maybe it's already documented somewhere,

No, it isn't. And to be honest, we didn't have a clear process in the past in the first place. I used to be the one to fix things with CrowdIn (with some help from Karin) although I am not very familiar with the tool and – as I do not use it very often – always have problems finding my way in the UI. Having said this, it would take some time for me to write this down and it would mean discussing some things to work out the process. I don't know when I will find the time for this.

Generally, you can do a lot with CrowdIn and have different roles so the process depends on who will do what. There are lots of general introductions to CrowdIn features based on user roles, see https://support.crowdin.com/. 

Is there someone else who could take over the manager role of the CrowdIn OER WM project? I would happily assist as far as my capabilities go when questions occur.

**Comment by gandalfar** [comment]
Ok, thanks. I'll discuss with @trugwaldsaenger who should do it and if it's me, where to put in on the timeline.

**Comment by tamielbr** [comment]
Ok, I will wait for orientation.
